src/Generator.Converter/Options.cs

```diff
@@ -159,43 +159,32 @@ internal OptionValueCollection (OptionContext c)
 			this.c = c;
 		}
 
-		#region ICollection
-		void ICollection.CopyTo (Array array, int index)  {(values as ICollection).CopyTo (array, index);}
+				void ICollection.CopyTo (Array array, int index)  {(values as ICollection).CopyTo (array, index);}
 		bool ICollection.IsSynchronized                   {get {return (values as ICollection).IsSynchronized;}}
 		object ICollection.SyncRoot                       {get {return (values as ICollection).SyncRoot;}}
-		#endregion
 
-		#region ICollection<T>
-		public void Add (string item)                       {values.Add (item);}
+				public void Add (string item)                       {values.Add (item);}
 		public void Clear ()                                {values.Clear ();}
 		public bool Contains (string item)                  {return values.Contains (item);}
 		public void CopyTo (string[] array, int arrayIndex) {values.CopyTo (array, arrayIndex);}
 		public bool Remove (string item)                    {return values.Remove (item);}
 		public int Count                                    {get {return values.Count;}}
 		public bool IsReadOnly                              {get {return false;}}
-		#endregion
 
-		#region IEnumerable
-		IEnumerator IEnumerable.GetEnumerator () {return values.GetEnumerator ();}
-		#endregion
+				IEnumerator IEnumerable.GetEnumerator () {return values.GetEnumerator ();}
```

Comment:
@Frassle In fact, looking at the source, this is a prevalent problem in a lot of source files. Perhaps it's better to perform a second cosmetic pass after this PR and replace the tabs with spaces everywhere.